common/src/test/java/com/microsoft/identity/common/BuildConfigTest.kt

```diff
@@ -31,11 +33,21 @@ class BuildConfigTest {
 
     @Test
     fun failIfBypassRedirecUriCheckEnabled(){
+        // Do not run this on scheduled test.
+        if (isBuildReason(BuildReason.Schedule)) {
+            return
+        }
```

Comment:
This is not the right way to skip test. You should use the JUnit Assume API. 

https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html#assumeFalse(boolean)

Same comment for all other places such pattern is being followed.